### PR TITLE
Kirilushkin Aleksandr Tasks 51-60

### PR DESCRIPTION
**Pull request by KirilushkinAleksandr**

### CODE REVIEW

File: tasks/54_roman-numbers-convert/roman-numbers-convert.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  num %= 50;
+  res.push(returnRomanNum(num / 10, 10));
+  num %= 10;
+  res.push(returnRomanNum(num / 5, 5));
+  num %= 5;
+  res.push(returnRomanNum(num, 1));
+  return res.join('');
+}
+
+function returnRomanNum(num, step) {
```
Comment (savchukm) [inline tasks/54_roman-numbers-convert/roman-numbers-convert.js]: `getRomanNum` is better name for function

File: tasks/54_roman-numbers-convert/roman-numbers-convert.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  num = Math.floor(num);
+  if (num === 4 || num === 9 || num === 5) {
+    step *= num;
+    num = 1;
+  }
+  if (num === 5) {
+    num = 1;
```
Comment (savchukm) [inline tasks/54_roman-numbers-convert/roman-numbers-convert.js]: This code block unachievable

File: tasks/55_find-object-analogy/find-object-analogy.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+function whatIsInAName(arr, searchValue) {
+  const res = [];
+  arr.forEach((Obj) => {
```
Comment (savchukm) [inline tasks/55_find-object-analogy/find-object-analogy.js]: Variable should have lower case first letter

**Review by savchukm — CHANGES_REQUESTED** [review]

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+function abbreviated(name) {
```
Comment (savchukm) [inline tasks/51_abbreviated-name/abbreviated-name.js:9]: Please jsdoc for every function

File: tasks/58_missed-letters/missed-letters.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+function findLetter(str) {
+  const lets = str.split('');
```
Comment (alexkond) [inline tasks/58_missed-letters/missed-letters.js]: Why you save letters? Are you afraid to run out of them? `letters` is a much better readable name.

**Review by alexkond — CHANGES_REQUESTED** [review]

**Review by alexkond — DISMISSED** [review]

**Review by KalachevDev — DISMISSED** [review]

**Review by alexkond — APPROVED** [review]

**Review by akimovaa — APPROVED** [review]

**Pull request merged**